.drone.yml

```diff
@@ -21,7 +21,7 @@ steps:
   name: identify-runner
 - commands:
   - make gen-go
-  - ./bin/grabpl gen-version --build-id ${DRONE_BUILD_NUMBER}
+  - ./bin/grabpl gen-version --source-commit ${DRONE_COMMIT}
```

Comment:
Will `DRONE_COMMIT` contain the short commit sha? In the documentation it looks like both the `DRONE_COMMIT_SHA` and `DRONE_COMMIT` contains the long version:

https://docs.drone.io/pipeline/environment/reference/drone-commit/
https://docs.drone.io/pipeline/environment/reference/drone-commit-sha/

Reply:
With this said. The documentation can be incorrect.

Reply:
It gets handled here https://github.com/grafana/build-pipeline/pull/377/files#diff-22c6a0ffc49e41334f68b289591713b9d1cc938580451cee934e6e9545148af3R25

Reply:
Aha, maybe better to do it in the grafana repository to keep it in the same place as we have lerna and can then be changed in a single PR in this repository. 

So maybe do something like: `${DRONE_COMMIT:0:10}` would be better?